CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/CameraFragment.kt

```diff
@@ -149,6 +151,14 @@ class CameraFragment : Fragment() {
         }
     }
 
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?): View? {
+        cameraExecutor = Executors.newSingleThreadExecutor()
```

Comment:
Can you move this to `onViewCreated`? I think it makes more sense there since `onCreateView` only inflates the layout.